metricbeat/module/vsphere/_meta/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG VSPHERE_GOLANG_VERSION
 FROM golang:${VSPHERE_GOLANG_VERSION}-alpine
 
 RUN apk add --no-cache curl git
-RUN go get -u github.com/vmware/govmomi/vcsim
+RUN go install github.com/vmware/govmomi/vcsim@latest
```

Comment:
Wouldn't it be better to pin the version so that we don't have this upgrading at arbitrary times?

Reply:
Depends on what we're testing here, the last available API or just that it works. Those are integration tests so I thought it would make sense to test against the last available simulator.

But I don't mind to pin it if you think it's necessary.

Reply:
I'd prefer to pin it to avoid the chance that some future breaking change or bug from this package doesn't break our tests.

Reply:
Pinned in https://github.com/elastic/beats/pull/35015/commits/b7127fd379ccc32b5691101e79560d7e5a617b4b